content/en/docs/instrumentation/net/libraries.md

```diff
@@ -89,12 +94,12 @@ app.Run();
 When you run this code and access the `/hello` endpoint, it will:
 
 * Start a new trace
-* Automatically generate a span representing the request made to the endpoint
-* Automatically generate a child span representing the HTTP GET made to
+* Generate a span representing the request made to the endpoint
+* Generate a child span representing the HTTP GET made to
```

Comment:
Similar to another issue, these spans are automatically generated on behalf of the user. Is it necessary to not say that here?

Reply:
instrumentation library created it - I will try improving the description tomorrow

Reply:
Addressed in https://github.com/open-telemetry/opentelemetry.io/pull/1565/commits/739e1298b8d9df6145d4dc33992f1b2f335c0e24